src/api/emulations/nfs.js

```diff
@@ -118,7 +118,12 @@ module.exports.open = (nfsHandle, fileHandle, openMode) => {
   return getObj(nfsHandle)
     .then((nfsObj) => getObj(fileHandle, true)
       .then((fileObj) => nfsObj.netObj.open(fileObj.netObj, openMode))
-      .then((file) => genHandle(nfsObj.app, file))
+      .then((file) => {
+        if (fileHandle) {
+          return replaceObj(fileHandle, nfsObj.app, file);
+        }
+        return genHandle(nfsObj.app, file);
```

Comment:
If we're returning the same file handle, let's update the docs above to make this clear.